modules/300-prometheus/templates/prometheus/prometheus.yaml

```diff
@@ -147,6 +147,7 @@ spec:
   ruleNamespaceSelector:
     matchLabels:
       heritage: deckhouse
```

Comment:
We should remove this label because `matchLabels` use `and` operator beetwen labels for checking.

And client need to add `heritage` label too. It is wrong. Client should use only one label.

Reply:
done